README.md

```diff
@@ -78,6 +82,10 @@ are supported as defined in [ISO/IEC 2022](https://en.wikipedia.org/wiki/ISO/IEC
 Each code sequence gets emitted with a dedicated event with its raw byte sequence:
 
 ```php
+<?php
+
+require __DIR__ . '/vendor/autoload.php';
+
```

Comment:
I don't think this adds a lot of value here given this is an incomplete snippet anyway, WDYT?

Reply:
Yes, thank you. 👍